pyfixest/FormulaParser.py

```diff
@@ -47,15 +54,15 @@ def __init__(self, fml):
             if "~" in fml_split[1]:
                 fevars = "0"
                 endogvars, instruments = fml_split[1].split("~")
-                # add endogeneous variable to "covars" - yes, bad naming
+                # add endogenous variable to "covars" - yes, bad naming
```

Comment:
Almost 10 years of econometrics and I still haven't learned how to write endogenous correctly ... 😅